packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -1206,17 +1200,18 @@ export function sendTransaction<
 							);
 						}
 					} catch (error) {
-						if (error instanceof ContractExecutionError) {
-							promiEvent.emit('contractExecutionError', error);
-						}
```

Comment:
I don't think there's a reason to have an `contractExecutionError` event when we already have a generic `error` event that every other error uses (@nikoulai please let me know if there was a specific reason for it since you added it in #5659)

Reply:
I see now that it's being using in [web3-eth-contract](https://github.com/web3/web3.js/blob/f6ff6ce0099c32c44b7e169d0bfdb4a1be842ca6/packages/web3-eth-contract/src/contract.ts#L1056)

Reply:
I've updated this code to check if `error instanceof InvalidResponseError || error instanceof ContractExecutionError` and emit `error` if so